### ISSUE TITLE
Support for projects with common.props?

### ISSUE THREAD

**Issue by antosubash** [versionize/versionize#80]
Thanks for this cool project. It helped me a lot.

The question I have is there any plan to support `common.props`. I used to manage version in the single place but now the version is mentioned in multiple places.

**Comment by skoniam** [comment]
For first step is good enough to add next mask to filters, eg.:
```csharp
   var filters = new[] { "*.vbproj", "*.csproj", "*.fsproj", "*.props" };
```
but in whole solucion names  like `projects` aren't a file projects.

**Comment by antosubash** [comment]
@skoniam Thanks for letting me know. I just created a PR by updating the filter.

**Comment by cabauman** [comment]
Fixed by #85
